open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/server/services/SubjectAreaGlossaryRESTServices.java

```diff
@@ -331,4 +321,68 @@ public SubjectAreaGlossaryRESTServices() {
         }
         return response;
     }
+
+    /**
+     * Get Glossary terms
+     *
+     * @param serverName serverName under which this request is performed, this is used in multi tenanting to identify the tenant
+     * @param userId     unique identifier for requesting user, under which the request is performed
+     * @param guid       guid of the category to get terms
+     * @return A list of terms for the glossary
+     * when not successful the following Exception responses can occur
+     * <ul>
+     * <li> UserNotAuthorizedException           the requesting user is not authorized to issue this request.</li>
+     * <li> InvalidParameterException            one of the parameters is null or invalid.</li>
+     * <li> PropertyServerException              Property server exception. </li>
+     * </ul>
+     * */
+    public SubjectAreaOMASAPIResponse<Term> getGlossaryTerms(String serverName, String userId, String guid) {
```

Comment:
Needs paging options - this call particularly is exposed here as there could be 1000s of terms in a large Glossary